batch/batches/StorageUpdate/KAsyncStorageUpdate.php

```diff
@@ -238,7 +267,14 @@ public function handleUsage($percent, $partner, $systemPartnerConfiguration, $mo
 				}
 				else
 				{
-					$this->systemPartnerClientPlugin->systemPartner->updateStatus($partner->id, KAsyncStorageUpdateUtils::PARTNER_STATUS_CONTENT_BLOCK, $reason);
+					try
+					{
+						$this->systemPartnerClientPlugin->systemPartner->updateStatus($partner->id, KAsyncStorageUpdateUtils::PARTNER_STATUS_CONTENT_BLOCK, $reason);
+					}
+					catch (Exception $e)

```

Comment:
Inside new functions - 
partnerUpdateStatus and partnerUpdateConfiguration
You should handle only relevant exceptions of type - 
catch(KalturaException $kex)
			{
   //exception raised by the server who responded
			}
			catch(KalturaClientException $kcex)
			{
//Exceptions raised by batch client
			}